### ISSUE TITLE
 make cannot find  -lp11-kit  while the files in /usr/lib64

### ISSUE THREAD

**Issue by 0x1orz** [cancerit/cgpBigWig#19]
```
curl -sSL --retry 10 https://github.com/cancerit/cgpBigWig/archive/0.4.2.tar.gz -o  cgpBigwig.tar.gz 
tar --strip-components 1 -C distro -xzf cgpBigwig.tar.gz  
make -C distro/c clean 
make -C distro/c -j$CPU prefix=$INST_PATH HTSLIB=$INST_PATH/lib LFLAGS="-L$INST_PATH/lib -L/usr/lib64 -L/usr/lib  -L$MySOs/lib "
```

Get the errer :   /usr/bin/ld: cannot find -lp11-kit
But I can find the files in /usr/lib64:
libp11-kit.so.0
libp11-kit.so.0.0.0
p11-kit
p11-kit-proxy.so

**Comment by ghost** [comment]
I suspect creating a symlink in /usr/lib64 to libp11-kit.so would fix this:

`ln -s /usr/lib64/libp11-kit.so.0 /usr/lib64/libp11-kit.so`

**Comment by 0x1orz** [comment]
Thx.Yeah, it works !

**Comment by ghost** [comment]
After some deeper investigation and trying to create a docker container for cgpBigWig (coming soon as dockstore-cgobigwig), it seems that Ubuntu 14.04 has a couple of libraries used by libBigWig available, whereas 16.04 has the numbered .so files available but no links. I had to `apt-get install` the p11-kit library and some others to get 16.04 to work.
